src/Compilers/Core/CodeAnalysisTest/StrongNameProviderTests.cs

```diff
@@ -32,61 +29,62 @@ public void ResolveStrongNameKeyFile()
             var provider = new VirtualizedStrongNameProvider(
                 existingFullPaths: fs,
                 searchPaths: ImmutableArray.Create(subdir));
```

Comment:
It looks like the `provider` instance can be replaced with a `VirtualizedStrongNameFileSystem` instance. At that point, if there are no other uses of `VirtualizedStrongNameProvider`, the class be removed.

Reply:
I tried removing this, but it tests some directory weirdness that VirtualizedStrongNameFileSystem doesn't support.  I'll file a bug to clean this up at a later date.

Reply:
@cston: filed here: https://github.com/dotnet/roslyn/issues/22327